sys/net/application_layer/gcoap/gcoap.c

```diff
@@ -42,7 +42,7 @@
 
 /* Internal functions */
 static void *_event_loop(void *arg);
-static void _listen(sock_udp_t *sock);
+static void _listen(coap_tl_sock_t *sock);
```

Comment:
This declaration must be below the definition of `coap_tl_sock_t`.

Reply:
Fixed this in https://github.com/RIOT-OS/RIOT/pull/12104/commits/236732f3224bbc7aeabf301b92d94aed7e2354ce